SDWebImage/SDImageCache.h

```diff
@@ -14,36 +14,31 @@
 
 typedef NS_OPTIONS(NSUInteger, SDImageCacheOptions) {
     /**
-     * By default, we do not query image data when the image is already cached in memory. This mask can force to query image data at the same time. However, this query is asynchronously unless you specify `SDImageCacheQueryMemoryDataSync`
+     * By default, we do not query image disk data when the image is already cached in memory. This mask can force to query image disk data only. However, this query is asynchronously unless you specify `SDImageCacheQueryDiskDataSync`
      */
-    SDImageCacheQueryMemoryData = 1 << 0,
-    /**
-     * By default, when you only specify `SDImageCacheQueryMemoryData`, we query the memory image data asynchronously. Combined this mask as well to query the memory image data synchronously.
-     */
-    SDImageCacheQueryMemoryDataSync = 1 << 1,
+    SDImageCacheForceQueryDiskData = 1 << 0,
```

Comment:
Acutally we can't use 2 options to handle 6 cases using permutations. Because 2! == 4